spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -27,12 +27,12 @@ module.exports = function (provider, setup) {
         console.warn('written');
         written = true;
       });
-      setTimeout(function () {
+      dispatch.gotMessageAsync('onData', [], function(msg) {
+        console.warn('received onData');
```

Comment:
I'd be okay with removing it, it is similar to the console.warns above and a few in the third test. Could also switch to log if that's preferred, not sure why it was originally warn (main effect seems to be that if you pipe the output to a logfile the warns just appear on your screen regardless). Anyway, if I remove or change it I'd probably remove/change all of them so let me know thoughts, and I think Will wrote this so he may have thoughts too.